src/hook/useRecruitListQuery.ts

```
@@ -0,0 +1,37 @@
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import api from '@hook/api';
+import { useFilterStore } from '@store/filterStore';
+import { useShallow } from 'zustand/react/shallow';
+
+export const useRecruitListQuery = <TData = any>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)